bindings/rust/wasmedge-sdk/src/config.rs

```diff
@@ -1,14 +1,27 @@
 use super::wasmedge;
 
-#[derive(Debug, Default)]
+#[derive(Debug)]
 pub struct Config {
     pub(crate) inner: wasmedge::Config,
 }
-
 impl Config {
-    pub fn with_wasi() -> Self {
-        let config = Self::default();
-        let inner: wasmedge::Config = config.inner.enable_wasi();
-        Self { inner }
+    pub fn create() -> Option<Self> {
```

Comment:
May we use `Result` for this return value?  Thanks.

Reply:
Accepted. Thanks!